features/column-pseudo.yml

```
@@ -0,0 +1,4 @@
+name: ::column pseudo-element
+description: The `::column` CSS pseudo-element represents the individual columns in a multi-column container and can snapped to and given associated `::scroll-marker` pseudo-elements.
```

```suggestion
description: The `::column` CSS pseudo-element represents the individual columns of a multi-column layout container and can snapped to and given associated `::scroll-marker` pseudo-elements.
```

I don't understand the second part "and can be snapped to ...". Can you please elaborate what you mean here? I read through the spec a little, but also got confused.

---

I've tried to clear this up. In particular, there are a limited set of properties supported on `::column` in the spec https://drafts.csswg.org/css-multicol-2/#column-pseudo:
> The [::column](https://drafts.csswg.org/css-multicol-2/#selectordef-column) pseudo-element accepts only a very limited set of properties:
>
> [scroll-margin](https://www.w3.org/TR/css-scroll-snap-1/#propdef-scroll-margin), [scroll-snap-align](https://www.w3.org/TR/css-scroll-snap-1/#propdef-scroll-snap-align), and [scroll-snap-stop](https://www.w3.org/TR/css-scroll-snap-1/#propdef-scroll-snap-stop) (the scroll snap properties that apply to elements inside a [scroll container](https://www.w3.org/TR/css-overflow-3/#scroll-container))

These properties are to allow individual columns to be snapped to by its scrolling container.